.env

```diff
@@ -0,0 +1,8 @@
+MONGODB_EXPOSED_PORT=27017
+MONGODB_ADMIN_USER=admin
+MONGODB_ADMIN_PASS=adminP4ss
```

Comment:
Generally not good idea to store passwords in the repo, it's better to ask user to provide one. But it seems fine as default I guess, depends on the purpose of this project :)

Reply:
this is a sample file for skeleton, so we can keep it

Reply:
@markelog the problem is that I can't use custom file name for .env file, f.e.: `.env.example`, maybe you know smth about this